engine/mp3juices.py

```diff
@@ -0,0 +1,47 @@
+
+from engine.root import BaseEngine
+class Mp3Juices(BaseEngine):
+
+    api_path = 'api/search.php'
+
+    def __init__(self):
+        super().__init__()
+        self.site_uri = "https://myfreemp3juices.cc/"
+        self.request_method = self.POST
+
+    def get_query_params(self,query=None,page=None,**kwargs):
+        if kwargs.get("id"):
+            return dict(id=kwargs.pop(id))
+        return  dict(q=query,page=page)
+
+    def get_url_path(self, page=None, category=None):
+        return (self.api_path)
+
+    def parse_parent_object(self,json,**kwargs):
```

Comment:
What does this function do

Reply:
line 17?

Reply:
its meant to return the api_path... the root engine's format_url func calls it... so a value is required.

Reply:
well it extracts necessary links from the json file and lets parse-single-object handle them